source/compatibility.tex

```diff
@@ -2225,13 +2225,13 @@
 \end{codeblock}
 
 \rationale
-In \Cpp{}, implicit int creates several opportunities for
+In \Cpp{}, implicit \tcode{int} creates several opportunities for
 ambiguity between expressions involving function-like
 casts and declarations.
 Explicit declaration is increasingly considered
 to be proper style.
 Liaison with WG14 (C) indicated support for (at least)
-deprecating implicit int in the next revision of C.
+deprecating implicit \tcode{int} in the next revision of C.
```

Comment:
I think we can say something more current here; it's been a long time since C was considering deprecation.

@AaronBallman What's the status quo with C17 and C2x? Can you suggest something suitable?

Reply:
Implicit `int` was removed in C99 (WG14 N635 N661 N692 N722), so at this point, we can just say it's incompatible with C.

Reply:
So, we should just drop this entire entry: There is (no longer) any incompatibility, because implicit int is not a thing in (current) C and not in C++. We refer to ISO 9899:2018 in the front matter, btw.

Reply:
"implicit int" will be removed by #5088